### DISCUSSION TITLE
VYPER

### DISCUSSION THREAD

**Discussion by Optimus523** [Cyfrin/moccasin-full-course-cu#19] [General]
Hope vyper is installable on windows 10. I opted to learn vyper before solidity along my journey to becoming a smart contract developer on Ethereum blockchain.

**Comment by Scofield-Idehen** [comment]
Yes, Vyper can be installed on Windows, there is a whole section of the course just for that. 

Best of luck

**Reply by Optimus523** [reply]
Thank you.
